### DISCUSSION TITLE
Error while running './create_forcing.exe namelist.input.ERA5': Fortran Runtime Error in create_forcing.f90

### DISCUSSION THREAD

**Discussion by ZRYings** [NCAR/hrldas#102] [Q&A]
Hello, I am encountering an issue when trying to run ./create_forcing.exe namelist.input.ERA5 and would appreciate any help to troubleshoot this error.

Error Message:
<img width="735" alt="image" src="https://github.com/NCAR/hrldas/assets/124751252/f4e54581-8fa4-4acb-893f-acf39088d351">

Context:
(1) I am using the ERA5-land global dataset that spans [60S, -180, 90N, 180]. I have created a corresponding domain with WPS (please see attachment 1 for namelist.wps).
(2) During the compilation process, I used GNU+MPI, with GFortran and GCC version 9.4.0.

I've already tried modifying the 'Name' for 'geopotential' in the Vtable to 'z' or 'TERRAIN', as suggested in previous threads, but the error persists. I apologize for opening a new thread, but I've gone through past discussions and couldn't find a solution to my issue. I have attached relevant data and files for reference. Thank you in advance for any help you can provide!
[forcing.zip](https://github.com/NCAR/hrldas/files/12567291/forcing.zip)

**Comment by tslin2** [comment]
Hi,
Please check and make sure all downloaded climate longitudes are between 0 and 360 degrees which has the same lat/lon as shown in geopotential and land mask NetCDF files. This is the default download in the website as Geographical area is set to Whole available region:
https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-land?tab=form

**Comment by sudarsan3010** [comment]
Hi @ZRYings,

how did you rectify this error?!
